src/lib/components/FilterForm.tsx

```diff
@@ -44,7 +56,7 @@ export const FilterForm: React.FC<IFilterFormProps> = () => {
   const setExpertsFilter = useCallback(
```

Comment:
this is fine just now, but consider moving this callback (as well as the one in effect hook) to the parent component, so that this filter form doesn't dictate the type of reducer arguments, as it might change in the future